File: 0009-palindrome-number/solution.java
class Solution {
    public boolean isPalindrome(int x)
    {
        if(x<0)
        {
            return false;
        }
        else
        {
            int n=x;
            int rev=0,rem;
            while(x>0)
            {
                rem=x%10;
                x=x/10;
                rev=rev*10+rem;
            }
            if(rev==n)
            {
                return true;
            }
            else
            {
                return false;
            }
        }
        
    }
}

File: 0009-palindrome-number/solution.c
bool isPalindrome(int x) {
    int y=x;
    long int sum=0;
    if(x<0)
    {
        return false;
    }
    while(x!=0)
    {
        int rem=x%10;
        sum=sum*10+rem;
        x=x/10;
    }
    if(sum==y)
    {
        return true;
    }
    else{
        return false;
    }
    
}
